sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -884,17 +888,18 @@ class PerKey(ptransform.PTransform):
 
     @staticmethod
     def add_timestamp(element, timestamp=core.DoFn.TimestampParam):
-      _check_instance_type(KV[T, T], element)
+      _check_instance_type(Tuple[T, T], element)
       key, value = element
       return [(key, (value, timestamp))]
 
     def expand(self, pcoll):
       return (pcoll
               | core.ParDo(self.add_timestamp)
+              .with_output_types(Tuple[T, Tuple[T, TimestampType]])
```

Comment:
Same here. I believe it should be KV[K, Tuple[T, TimestampType]]

Reply:
Sorry. My bad. I'll change it.